src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/ios-manual-installation.mdx

```diff
@@ -170,7 +170,7 @@ If you have trouble getting your app to crash, the New Relic agent provides an A
 
 ## Optional: Change the logging level [#logging]
 
-By default the New Relic agent will log at the `info` level. Increasing the log level to `verbose` or higher is only encouraged for debugging and not for release builds. 
+By default the New Relic agent will log at the `info` level. Increasing the log level to `verbose` or higher priority is only encouraged for debugging and not for release builds. Verbose level or higher priority will log all API interactions with the New Relic server. 
```

Comment:
Can you please add this change to the other places in the docs where we warn about verbose priority level or higher?

Reply:
I added it to the two tvos pages.